### PR TITLE
changes some CAS cards to be yogs friendly

### PR DESCRIPTION
**Pull request by ktlwjec0**
some very lewd naughty cards that were never changed, there are a few still there but we can do those another time when we have motivation again to finish this

:cl:  Ktlwjec, Breadtoucher
tweak: Makes CAS cards yogs friendly.
/:cl:

### CODE REVIEW

**Comment by Moja10** [comment]
Sad... some of these are really funny

File: strings/cas_white.txt [hunk 1]
```diff
@@ -141,7 +141,7 @@ Ash drakes.
 A fascist captain.
 FREE DRONE.
 Memes.
-Warm, velvety lizard sex.
+Marrying a lizard in the chapel.
```
Comment (solwashere) [inline strings/cas_white.txt]:
```suggestion
Marrying a vuulek in the chapel.
```

File: strings/cas_white.txt [hunk 2]
```diff
@@ -151,27 +151,27 @@ Immulsions.
 100 potency bluespace bananas.
 The crotchety IBM piece of shit you're supposed to call an AI.
 An irritatingly chipper borg.
-The lizard fuckpile.
+A lizard clique that refuses to speak to non-lizards.
```
Comment (solwashere) [inline strings/cas_white.txt]:
```suggestion
A vuulek clique that refuses to speak to non-vuulen.
```

**Review by solwashere — COMMENTED** [review]
cough

File: strings/cas_white.txt (hunk 2 above)
Comment (TheRyeGuyWhoWillNowDie) [reply strings/cas_white.txt]: the fuck is a vuleek

Comment (JohnFulpWillard) [reply strings/cas_white.txt]: lore writers, not even once.

Comment (UltraBigRat) [reply strings/cas_white.txt]: considering this is is a  funny card game, why refer to them with a formal name no one remembers?

Comment (solwashere) [reply strings/cas_white.txt]: why does it matter that they not get called by their formal names? They very much are in other areas, i.e. traitor objectives, so why not make it more consistent.

Comment (ktlwjec0) [reply strings/cas_white.txt]: i dont agree but i'll change it because i dont care enough to argue

**Review by ToasterBiome — APPROVED** [review]
my soul!!!

**Comment by ktlwjec0** [comment]
checks needs rerunning :(

**Review by ynot01 — APPROVED** [review]
Makes a dead card game even worse, but yes this does not belong in Yogs

**Pull request merged**